sdk/monitor/azure-monitor-opentelemetry-exporter/src/test/java/com/azure/monitor/opentelemetry/exporter/AzureMonitorExportersEndToEndTest.java

```diff
@@ -241,15 +240,45 @@ public void testStatsbeat() throws Exception {
         validateFeatureStatsbeat(featureStatsbeat);
     }
 
+    @Test
+    public void testStatsbeatShutdownWhen400InvalidIKeyReturned() throws Exception {
+        String fakeBody = "{\"itemsReceived\":1,\"itemsAccepted\":0,\"errors\":[{\"index\":0,\"statusCode\":400,\"message\":\"Invalid instrumentation key\"}]}";
+        MockedHttpClient mockedHttpClient =
+            new MockedHttpClient(
+                request -> {
+                    return Mono.just(new MockHttpResponse(request, 400, new HttpHeaders(), fakeBody.getBytes()));
+                });
+
+        // create OpenTelemetrySdk
+        CountDownLatch countDownLatch = new CountDownLatch(1);
+        AzureMonitorExportersEndToEndTest.CustomValidationPolicy customValidationPolicy = new AzureMonitorExportersEndToEndTest.CustomValidationPolicy(countDownLatch);
+        OpenTelemetrySdk openTelemetrySdk =
+            TestUtils.createOpenTelemetrySdk(
+                getHttpPipeline(customValidationPolicy, mockedHttpClient), getConfiguration(), STATSBEAT_CONNECTION_STRING);
+
+        generateMetric(openTelemetrySdk);
+
+        // close to flush
+        openTelemetrySdk.close();
+
+        Thread.sleep(1000);
+
+        // wait for export
+        countDownLatch.await(10, SECONDS);
+        assertThat(customValidationPolicy.url)
+            .isEqualTo(new URL("https://westus-0.in.applicationinsights.azure.com/v2.1/track"));
+        assertThat(customValidationPolicy.actualTelemetryItems.stream().filter(item -> item.getName().equals("Statsbeat")).count()).isEqualTo(0);
+    }
+
     private static Map<String, String> getConfiguration() {
-        return Collections.singletonMap("APPLICATIONINSIGHTS_CONNECTION_STRING", CONNECTION_STRING_ENV);
+        return Collections.singletonMap("APPLICATIONINSIGHTS_CONNECTION_STRING", STATSBEAT_CONNECTION_STRING);
     }
 
     private static Map<String, String> getStatsbeatConfiguration() {
         Map<String, String> map = new HashMap<>(3);
         map.put("APPLICATIONINSIGHTS_CONNECTION_STRING", CONNECTION_STRING_ENV);
-        map.put("STATSBEAT_LONG_INTERVAL_SECONDS", "1");
-        map.put("STATSBEAT_SHORT_INTERVAL_SECONDS", "1");
+        map.put("STATSBEAT_LONG_INTERVAL_SECONDS_PROPERTY_NAME ", "1");
+        map.put("STATSBEAT_SHORT_INTERVAL_SECONDS_PROPERTY_NAME ", "1");
```

Comment:
```suggestion
        map.put("STATSBEAT_LONG_INTERVAL_SECONDS_PROPERTY_NAME", "1");
        map.put("STATSBEAT_SHORT_INTERVAL_SECONDS_PROPERTY_NAME", "1");
```

Reply:
when i copy and paste, it always add an extra space, do you know why? this is my second time